### PR TITLE
Pooja

### PR DESCRIPTION
**Pull request by ManisshaMalapati**

### CODE REVIEW

**Comment by ps11061601** [comment]
> _No description provided._

**Review by chrisnellis — APPROVED** [review]

**Review by chrisnellis — APPROVED** [review]

**Review by ps11061601 — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -20,6 +29,18 @@ const routes: Routes = [
     data: {
       onAuthRequired
     }
+  },
+  {
+    path: 'student-login',
+    component: StudentHomePageComponent
```
Comment (chrisnellis) [inline src/app/app-routing.module.ts:42]: If you want these pages to be protected by login, you can use:
`canActivate: [OktaAuthGuard],` in the route. 

That way, you can't access those pages without logging in first. I'd suggest you add them before merging

**Review by chrisnellis — CHANGES_REQUESTED** [review]
Please add the guards before merging

**Comment by chrisnellis** [comment]
You can check this branch out [here](https://lightbulb-lms-front-end-git-pooja.lightbulb-lms.vercel.app/)

**Review by chrisnellis — APPROVED** [review]

**Pull request merged**